src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,94 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class RegistrationServiceImplTest {
+    private static User expected;
+    private static User exist;
+    private static RegistrationServiceImpl registrationService;
+    private User actual;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        expected = new User();
+        expected.setAge(65);
+        expected.setLogin("qwerty@gmail.com");
+        expected.setPassword("123456");
+    }
+
+    @BeforeEach
+    void setUp() {
+        actual = new User();
+    }
+
+    @Test
+    void register_userValid_Ok() {
+        actual.setAge(65);
+        actual.setLogin("qwerty@gmail.com");
+        actual.setPassword("123456");
```

Comment:
can we initialize this object once with default values and then in test methods change them if we need to?